packages/js/src/ui/components/Header/StatusDropdown.tsx

```diff
@@ -1,22 +1,16 @@
-import { NotificationStatus } from '../../../types';
 import { useInboxStatusContext } from '../../context';
-import { cn, useStyle } from '../../helpers';
+import { useStyle } from '../../helpers';
 import { ArrowDropDown } from '../../icons';
 import { Dropdown, dropdownTriggerVariants } from '../Dropdown';
 import { StatusOptions } from './StatusOptions';
 
-/**
- *
- * TODO: Implement getStatusLabel after Filter is implemented
- * currently, it is a placeholder function
- */
-const getStatusLabel = (status?: NotificationStatus) => {
+const getStatusLabel = (status?: { read?: boolean; archived?: boolean }) => {
   switch (status) {
-    case NotificationStatus.UNREAD:
+    case { read: undefined, archived: false }:
       return 'Inbox';
-    case NotificationStatus.UNSEEN:
+    case { read: false, archived: false }:
       return 'Unread';
-    case NotificationStatus.SEEN:
+    case { read: true, archived: false }:
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** please update this switch according to the suggestions below in the StatusOptions